osu.Game/Rulesets/Replays/Types/IConvertibleReplayFrame.cs

```diff
@@ -18,5 +18,11 @@ public interface IConvertibleReplayFrame
         /// <param name="beatmap">The beatmap.</param>
         /// <param name="lastFrame">The last post-conversion <see cref="ReplayFrame"/>, used to fill in missing delta information. May be null.</param>
         void ConvertFrom(LegacyReplayFrame currentFrame, IBeatmap beatmap, ReplayFrame lastFrame = null);
+
+        /// <summary>
+        /// Populates this <see cref="ReplayFrame"/> using values from a <see cref="LegacyReplayFrame"/>.
+        /// </summary>
+        /// <param name="beatmap">The beatmap.</param>
+        LegacyReplayFrame ConvertTo(IBeatmap beatmap);
```

Comment:
Please rename to `ToLegacy`, and the one above to `FromLegacy`. It's very weird naming as is.